src/chainList.ts

```diff
@@ -0,0 +1,57 @@
+import { RpcConfig } from './typings'
+
+export enum Chains {
```

Comment:
I've mostly seen a [preference for singular names](https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural) for enums instead of plurals. Singular names work better in type signatures and don't imply a collection type.

```
const getConfigFromChain = (c: Chain): RpcConfig =>
// instead of
const getConfigFromChain = (c: Chains): RpcConfig =>
```

Reply:
@ajithranka Updated.